geequel-shell/src/test/java/org/neo4j/shell/state/BoltStateHandlerTest.java

```diff
@@ -94,9 +83,9 @@ public Driver apply(String uri, AuthToken authToken, Config config) {
                 super.apply(uri, authToken, config);
                 return new FakeDriver() {
                     @Override
-                    public Session session(AccessMode accessMode, String bookmark) {
-                        return new FakeSession();
-                    }
+                   public Session session() {
+                       return new FakeSession();
+                   }
```

Comment:
I'm not actually certain about the indentation here. It's possible you want to add one more space...